pkg/generator/key/key.go

```diff
@@ -14,4 +14,5 @@ const (
 
 var (
 	ConfigVersion = project.Name() + ".giantswarm.io/config-version"
+	AppLabel      = project.Name() + ".giantswarm.io/app"
```

Comment:
Should we prefix those? Why not use `app.kubernetes.io/name`? Maybe we should copy all the labels from the reconciled App CR?

Reply:
This is a direct result of https://github.com/giantswarm/config-controller/blob/master/pkg/label/label.go#L10

Reply:
Hmm. Copy-pasting annotations and labels seems ok, but when we generate using CLI we don't have that luxury. I can overwrite with labels/annotations from the App CR in handlers though.